src/main/webapp/assets/tpl/editor/page.html

```diff
@@ -47,12 +47,12 @@ <h3 style="margin-top: 6px;">Blueprint Composer</h3>
                     </span>
                 </div>
             </div>
+            <div class="error-message label-message hide" style="display:none;">
+                <div class="label-important">ERROR</div>
+                <span class="error-message-text"></span>
+            </div>
             <div class="navbar_main_wrapper">
                 <div class="navbar_main">
-                    <div class="error-message label-message hide" style="display:none;">
```

Comment:
@bostko It should be part of this PR in my opinion as this is a specific style, not a global one. You just need to add the code I mentioned to the `base.css` file :)